src/LayerTreeNode/LayerTreeNode.less

```diff
@@ -0,0 +1,3 @@
+.react-geo-layertree-node {
+
+}
```

Comment:
Plz remove this file if it is not used.

Reply:
I agree, but as the `LayerTreeNode` component is a dummy component at the moment, I would leave it here for reference.